rotkehlchen/globaldb/handler.py

```diff
@@ -237,6 +237,20 @@ def cleanup(self) -> None:
         if self._packaged_db_conn is not None:
             self._packaged_db_conn.close()
 
+    @staticmethod
```

Comment:
I don't understand why you are making it a static method and then you are passing the connection? And the connection you pass in all cases is the globaldb own connection. 

This means this should be a normal method with no arguments and use `self.conn` inside

Reply:
in both cases where I use it I already have the globaldb object initialized by `GlobalDbHandler()` and It takes the conn for that initailized instance